### PR TITLE
DCA-8 create prod account for DCA

### PR DESCRIPTION
**Pull request by brucehoff**
This PR creates the AWS account to host the DCA Shiny application.

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -566,6 +567,21 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  DCAProdAccount:
```
Comment (zaro0508) [inline org-formation/organization.yaml:570]: i don't see a DCA dev account.  was that intentional?

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -566,6 +567,21 @@ Organization:
...
+      RootEmail: aws-dca-prod@sagebase.org
+      Alias: org-sagebase-dca-prod
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: FAIR
+        Project: DCA
+        AccountOwner: milen.nikolov@sagebase.org
+        CostCenter: NO PROGRAM / 000000
+        budget-alarm-threshold: 1000
+        budget-alarm-threshold-email-recipient: aws-dca-prod@sagebase.org
```
Comment (zaro0508) [inline org-formation/organization.yaml:583]: i don't see @milen-sage or any other scientists as a member of this group.  is that intentional?

File: org-formation/organization.yaml [hunk 3]
```diff
@@ -566,6 +567,21 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  DCAProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: dca-prod
```
Comment (zaro0508) [inline org-formation/organization.yaml:573]: this should be the same as the alias

**Review by zaro0508 — CHANGES_REQUESTED** [review]
still no issue tied to this PR?

File: org-formation/organization.yaml (hunk 1 above)
Comment (brucehoff) [reply org-formation/organization.yaml:570]: Yes. Milen's team is using the D&T-dev account.

**Comment by brucehoff** [comment]
> still no issue tied to this PR?

The Jira issue is DCA-8.  I updated the issue title to include the issue ID.

File: org-formation/organization.yaml (hunk 2 above)
Comment (brucehoff) [reply org-formation/organization.yaml:583]: Good point.  I added Milen to the Google Group.

File: org-formation/organization.yaml (hunk 3 above)
Comment (brucehoff) [reply org-formation/organization.yaml]: done!

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**